### DISCUSSION TITLE
Is there any way to convert from cmd an exr to png (or similar) using oiio?

### DISCUSSION THREAD

**Discussion by lgander** [OpenImageIO/oiio#3579] [Q&A] [answered]
Today I installed vcpkg and then OIIO.

I'm looking for a way to quickly convert EXR files to PNG (or JPG maybe). Let me explain:

I work in VFX (continuously dealing with exr files) and I'm using Seer in windows, which is like Mac OS QuickLook: you hit space bar over a file and it shows a preview of the file.
It doesn't preview EXR files natively, but it has a plugin that convert EXR to PNG using ImageMagick and then previews that PNG. The problem is that its slow and it doesn't work with ACES. 

So I'm looking for an alternative way to properly convert the EXR files so I can fast and accurately preview them by hitting the space bar.

Is there any way to accomplish this task from cmd using oiio ?
Any other ideas are very welcome.
Thanks!

**Comment by lgritz** [comment, marked as answer]
Just straightforwardly moving the data in the exr file to png or jpeg would look like

    oiiotool in.exr -o out.png

But you mentioned something about ACES, so I'm thinking that the exr file is known to be in ACES and of course you probably want the PNG or JPEG to properly be in sRGB, right? So here is where things get complicated. If you have OpenColorIO support built in, and an OCIO config that defines ACEScg and sRGB color spaces, then you can also do the color space transformation like this:

    oiiotool in.exr --colorconvert ACEScg sRGB -o out.png

**Reply by lgander** [reply]
Thanks for your help.

I´ve read about oiiotool before installing oiio, and i was aiming to use it. 
But after installing oiio I couldn´t find any executable of oiiotool and concluded that it was a lib that can´t be run straight from cmd 

Maybe I am missing something very basic here?

**Reply by lgander** [reply]
I've downloaded a zip containing the oiiotool.exe file from this link: https://www.patreon.com/posts/openimageio-oiio-53939451
It works really good. This is what I was looking after: 
oiiotool.exe  in.exr  --tocolorspace sRGB -o out.png

Anyway, I would like to know if the **oiiotool.exe** should been already installed when I've installed oiio using vcpkg. Just to know if I've done something wrong during installation and learn from this.

Or maybe the .exe was not supposed to be there and the one i've downloaded was compiled by someone specifically to be run from cmd?

**Reply by lgritz** [reply]
Indeed! Vcpkg only installs the libraries by default. Other parts install only when you invoke certain options. As I understand it, to get the command line tools, you need

    vcpkg install openimageio [tools]

For a full list of supported build features: `vcpkg search openimageio`

**Reply by lgander** [reply]
Thanks @lgritz !
